assembl/configs/base_env.yaml

```diff
@@ -186,3 +186,8 @@ attachment_allowed_mime_types:
   - image/*
   - video/*
   - application/pdf
+  - application/msword
+  - application/vnd.openxmlformats-officedocument.wordprocessingml.document
+  - application/vnd.ms-powerpoint
+  - application/vnd.openxmlformats-officedocument.presentationml.presentation application/vnd.ms-excel
+  -	application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
```

Comment:
The last two points are weird. Don't you need one mimetype per line?

Reply:
Yes, I missed a newline, and there was a tab instead of space, I didn't catch it on atom. Fixed.